models/user.ts

```diff
@@ -5,16 +5,16 @@ class User{
     name: string;
```

Comment:
Could change the variable name from 'name' to 'fullName' which would be easy to understand

Reply:
Resolved